### ISSUE TITLE
Question: are there cases when the property condition shouldn't be "CONFLICT" but should be tool-specific?

### ISSUE THREAD

**Issue by jhmann** [datascience/c3po#33]
When initially discussing these rules, my understanding is that the conditions were more specific: if this tool identifies this property as this, and this other tool identifies the same property as that, then pick this one. This seems to have been simplified to: if this property is conflicted and given these other conditions, then use the selected property. Is this simplified algorithm going to correctly identify and cover all needed cases? Without comparing the conflicted metadata for a larger sample (which I can't do easily until issue #16 is resolved) I can't tell.

**Comment by artourkin** [comment]
Yes, the current interface to create rules doesn't allow to address complicated cases. It requires improvements. This is related to the issue #32.
